tests/util/TestServerConfig.ts

```diff
@@ -4,6 +4,9 @@ import { GameMapType } from "../../src/core/game/Game";
 import { GameID } from "../../src/core/Schemas";
 
 export class TestServerConfig implements ServerConfig {
+  origin(): string {
+    return "http://localhost:9000";
```

Comment:
The jest code doesn't listen on 9000, so it might be better to clarify that this is just a placeholder with something like:
```suggestion
    return "unused";
```